### ISSUE TITLE
Dividend Data Appears Erroneous

### ISSUE THREAD

**Issue by dontspyonme** [iexg/IEX-API#407]
## Summary
Thank you for making the API free to use. I was downloading dividend data and noticed what would appear to be a mistake in the historical dividend data for AT&T (ticker = T). The data from IEX is missing one of AT&T's historical dividend payments.

## Details
I used this: <https://api.iextrading.com/1.0/stock/T/dividends/5y> to pull historical dividend data for AT&T. The returned dividend data is missing the November 1st, 2016 (payment data) $0.48 / share dividend for AT&T.

### Date/time of issue
July 1st, 2018

### API URL
https://api.iextrading.com/1.0/stock/T/dividends/5y

### What you expected to happen
To get all dividends for the past 5 years.

### What actually happened
Missing one dividend from 2016. See here for what I was expecting:
https://investors.att.com/stock-information/historical-stock-information/historical-dividends/att-inc

**Comment by rodoabad** [comment]
@iftiq did you notice that IEX dividend data is also outdated? Looks like updates stopped at around March.

**Comment by sdvisconti** [comment]
Numerous users, including myself, have been reporting for weeks that
dividend and split information has errors and omissions - most notably
there isn't any data beyond March 9, 2018.  I have not seen any response
addressing this concern.

Is the issue being researched and/or actively worked on?  Is there an ETA
for resolution?

Regards,
Scott

**Comment by rodoabad** [comment]
@sdvisconti looks like bug fixes in version `1.0` are put on hold because they are moving their resources to releasing `2.0`.

**Comment by joshuablackburn-iex** [comment]
This is related to our 3rd party provider.  We are working to figure out what is going on with the existing data, and in parallel, working to source a better provider.

**Comment by kotarofujita** [comment]
Is there any updates for this issue or do we need to wait for version 2.0 for this to work again?

**Comment by ashern** [comment]
Have y'all at IEX found a dividend source yet that's not Yahoo Finance, or is this API dead for good?

**Comment by evancarter-iex** [comment]
We will let you know when we have updates to this.

**Comment by ashern** [comment]
Thanks, @evancarter-iex. 

Your lack of Dividend data for the last 6 months is also [affecting prices that you disseminate](https://iextrading.com/developer/#faq).

I recognize that this is a free API, but IEX is presumably not a rag-tag group of volunteers. Hopefully this basic functionality gets addressed sooner than later.

**Comment by joshuablackburn-iex** [comment]
Is this not working for you?  https://iextrading.com/api/1.0/stock/aapl/dividends/6m

Our data is currently through a 3rd party and we are aware of the reported issues with the quality they are providing.  Like I've mentioned before, we are working with a new partner to bring the highest quality data on the market to the API.  This is a very unique partnership and has taken time to get it structured the way we want for our users.  This data is not cheap, and you are able to find an alternate source in the meantime, but it will cost a considerable amount.  We are currently taking on this cost to provide this data for free.  We are working hard to get the next version out with better data.  Thank you for your patience on this.

Also worth noting, our historical prices are not affected by the lack of dividend data.  Historical prices are adjusted before coming to IEX.

**Comment by ashern** [comment]
As far as I can tell, no new dividends have been added since at least March of this year. The AAPL link you sent, for instance, is missing ex-dates 5/11/18 and 8/10/18.

If you take a look at the other bug report I linked, the lack of dividends is also causing you to report unadjusted daily change/previous close on ex-date.

**Comment by evancarter-iex** [comment]
Please see https://github.com/iexg/IEX-API/issues/557

**Comment by evancarter-iex** [comment]
This is now fixed in IEXCloud. Sign up for beta access, if you haven't already.

https://iexcloud.io/
